.github/workflows/ci_build_test.yml

```
@@ -46,4 +46,21 @@ jobs:
         with:
           command: fmt
           args: --verbose -- --check
-
+      # `clippy-check` will run `cargo clippy` on new pull requests. Due to a limitation in GitHub
+      # permissions, the behavior of the Action is different depending on the source of the PR. If the
+      # PR comes from the ion-rust project itself, any suggestions will be added to the PR as comments.
```

typo: `ion-rust` -> `partiql-lang-rust`
```suggestion
      # PR comes from the partiql-lang-rust project itself, any suggestions will be added to the PR as comments.
```

---

Good catch, resolved in the new commit.